mypy/suggestions.py

```diff
@@ -489,7 +489,7 @@ def find_node(self, key: str) -> Tuple[str, str, FuncDef]:
         else:
             target = split_target(self.fgmanager.graph, key)
             if not target:
-                raise SuggestionFailure("Cannot find module for %s" % (key,))
+                raise SuggestionFailure("Cannot find module type information for %s" % (key,))
```

Comment:
This is actually not relevant for #6513, since this change only applies to `dmypy suggest`, which is still an undocumented feature. Feel free to open a new issue if you still think that this is worth changing.

Reply:
oops, I blindly changed the only matching line in the code - sorry for wasting your time!

it seems the "fix" was already done in 70f2857f2ef867ce6d238838367406bc3e8e14de ;) never mind me then!